shared/src/packages/trucks/libs/types/truck-add-request-dto.type.ts

```diff
@@ -0,0 +1,28 @@
+import { type ValueOf } from '~/libs/types/types.js';
+
+import {
+  type TruckManufacturer,
+  type TruckTowType,
+  type TruckYear,
+} from '../enums/enums.js';
+
+type TruckAddRequestDto = {
+  manufacturer: {
+    label: string;
+    value: ValueOf<typeof TruckManufacturer>;
+  };
```

Comment:
not sure why labels are part of DTO

Reply:
this is no longer DTO this is the add-form model type